src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISApplicationConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Server.IISIntegration
```

Comment:
this is super ugly. I wanted to make a config file that is loaded once and available statically. Not sure what the best pattern is here.